universe/Meter.h

```diff
@@ -48,7 +48,7 @@ class FO_COMMON_API Meter {
         init = FromFloat(DEFAULT_VALUE);
     }
 
-    constexpr void AddToCurrent(float adjustment) noexcept { cur = FromFloat(FromInt(cur) + adjustment); }
+    constexpr void AddToCurrent(float adjustment) noexcept { cur += FromFloat(adjustment); }
```

Comment:
While not being totally wrong, it does not make sense to use fixed point arithmetic for storage only, while all the calculations are one by casting to float and back.